app/models/lesson.rb

```diff
@@ -20,6 +20,11 @@ class Lesson < ApplicationRecord
 
   attribute :completed, :boolean, default: false
 
+  def new?
```

Comment:
Nit: `new?` is a little too close to the method `new` that most ruby classes have. It'll work fine, but might cause a little bit of confusion and require a second glance when reading through the code.

We could go with something more expressive here:

```suggestion
  def recently_added?
```

or 
```suggestion
  def recently_created?
```